### PR TITLE
Add version history for 5.3.3

### PR DESCRIPTION
**Pull request by dgault**
See https://github.com/openmicroscopy/bioformats/milestone/47?closed=1
for list of included PRs.

### CODE REVIEW

File: docs/sphinx/about/whats-new.txt [hunk 1]
```diff
@@ -1,6 +1,66 @@
...
+File format fixes:
+
+* Applied Precision CellWorX
+   - fix to now display the correct plate name and dimensions
+
+* NIFTI
+   - a few fixes for problems with byte alignment when reading non-core 
+     metadata from NIFTI headers
+
+* LeicaLIF
```
Comment (rleigh-codelibre) [inline docs/sphinx/about/whats-new.txt]: Space before LIF?

File: docs/sphinx/about/whats-new.txt [hunk 2]
```diff
@@ -1,6 +1,66 @@
...
+   - added support for timestamps of lif files created with LAS AF 3.1 or
```
Comment (rleigh-codelibre) [inline docs/sphinx/about/whats-new.txt]: Capitalise lif, or use `.lif` with a dot and `:file:` markup?

File: docs/sphinx/about/whats-new.txt [hunk 3]
```diff
@@ -1,6 +1,66 @@
...
+   - for backwards compatibility an option to preserve pre-5.3.3 physical sizes 
+     has been added. This can be set either via command line tools, through 
+     the API with the DynamicMetadataOptions class, or in the Bio-Formats 
+     plugin configuration in ImageJ
+
+* Improvision TIFF
+   - channel colors are now being read and if present set correctly in image metadata
+
+* MetaMorph
+   - Fix for OutOfMemoryError exceptions when reading large Metamorph TIFF plates
```
Comment (rleigh-codelibre) [inline docs/sphinx/about/whats-new.txt]: lowercase `Fix` for consistency?

**Comment by rleigh-codelibre** [comment]
Looks fine, a few minor comments noted.  Not sure why maven is failing; could this be restarted @sbesson ?

**Comment by sbesson** [comment]
@rleigh-codelibre I am reviewing https://github.com/openmicroscopy/bioformats/pull/2741 which should fix the Travis Maven build on Bio-Formats

File: docs/sphinx/about/whats-new.txt [hunk 4]
```diff
@@ -1,6 +1,66 @@
...
+
+Bug fixes:
+
+* ImageJ
+   - fix for issue when exporting from an ImagePlus that represents signed 
+     data. The pixel type will now remain unchanged as will the pixel values 
+     which had previously been scaled incorrectly
+
+* Command line tools 
+   - fix for IllegalArgumentException when using bfconvert via command line
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: We've used mark up for classes etc in previous posts

File: docs/sphinx/about/whats-new.txt [hunk 5]
```diff
@@ -1,6 +1,66 @@
...
+   - added support for timestamps of LIF files created with LAS AF 3.1 or 
+     newer. In the case of a halted acquisition only non-null timestamps are 
+     stored in the OME metadata
+   - the physical pixel height and width were incorrectly calculated by 
+     dividing by the number of pixels. This has now been corrected to match 
+     the official Leica LIF specification documents by dividing by the number 
+     of pixels minus one
+   - for backwards compatibility an option to preserve pre-5.3.3 physical sizes 
+     has been added. This can be set either via command line tools, through 
+     the API with the DynamicMetadataOptions class, or in the Bio-Formats
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: Could use mark up for the class here too

File: docs/sphinx/about/whats-new.txt [hunk 6]
```diff
@@ -1,6 +1,66 @@
...
+   - for backwards compatibility an option to preserve pre-5.3.3 physical sizes 
+     has been added. This can be set either via command line tools, through 
+     the API with the DynamicMetadataOptions class, or in the Bio-Formats 
+     plugin configuration in ImageJ
+
+* Improvision TIFF
+   - channel colors are now being read and if present set correctly in image metadata
+
+* MetaMorph
+   - fix for OutOfMemoryError exceptions when reading large Metamorph TIFF plates
```
Comment (hflynn) [inline docs/sphinx/about/whats-new.txt]: Again, mark up

**Pull request merged**